src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,80 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import duke.exceptions.StorageException;
+import duke.parser.FileParser;
+import duke.tasks.Task;
+
+/**
+ * The Storage class handles the loading and saving of tasks to and from a file.
+ */
+public class Storage {
+    private final String filePath;
+
+    /**
+     * Constructs a Storage object with the specified file path.
+     *
+     * @param filePath the path of the file to be loaded/saved
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Creates a new file at the specified path if it does not exist.
+     *
+     * @param f the file to be created
+     * @throws StorageException if an error occurs while creating the file
+     */
+    private void createFile(File f) throws StorageException {
```

Comment:
Good way to name the method in this way! I noticed that this act is used elsewhere as well. good job!